db/seeds.rb

```diff
@@ -1,7 +1,62 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
-#
-# Examples:
-#
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
+require File.expand_path('../../lib/seed_helper.rb', __FILE__)
+require 'csv'
+require 'open-uri'
+require 'colorize'
+
```

Comment:
you are correct, though, that we need to add a rake task to destroy existing trains when it is time to refresh all of the train data, but not the user data.